### ISSUE TITLE
[DOCS] Bayesian Kernel Density 

### ISSUE THREAD

**Issue by koaning** [koaning/scikit-lego#290]
We might want to combine the GMM stuff with this under a `probabilistic` banner.

**Comment by nvanommeren** [comment]
Hee! Just to double-check, do you mean the documentation of the BayesianKernelDensityClassifier in neighbors.py?

**Comment by koaning** [comment]
Yes. It'd be cool to have it combined in some way here: https://scikit-lego.readthedocs.io/en/latest/mixture-methods.html.

**Comment by C3L0** [comment]
Hi @koaning! I see this issue is still open. I would like to contribute by adding the Bayesian Kernel Density documentation.

**Comment by koaning** [comment]
go right ahead, but please no ai full automation on that one. just a simple human-interpretable example plz
